src/Courses/CourseDetailsTabs/CourseDetailsTab.tsx

```diff
@@ -113,14 +115,32 @@ export const CourseDetailsTab: React.FC<CourseDetailsTabProps> = ({ course, load
                 {userType !== UserRole.STUDENT &&
                     <div
                         style={{
-                            marginLeft: 'auto'
+                            marginLeft: 'auto',
+                            display: 'flex',
                         }}
                     >
+                        { inEditMode &&
```

Comment:
Is there a reason to only show the Course Archive while in Edit Mode?

Reply:
Import is only in edit mode, I don't want it to just sit there all the time since it is not used often